.gitignore

```diff
@@ -7,3 +7,5 @@ bin
 gradle.properties
 *.db
 .sts4-cache/classpath-data.json
+.idea
```

Comment:
IntelliJ, eh?  Funny, Eclipse was causing some problems with unit tests for me today, and they worked fine in my coworker's intelliJ.  Maybe not such a bad idea after all.